sdk/src/memif_impl.c

```diff
@@ -26,7 +26,7 @@ void print_memif_details(memif_conn_handle_t conn)
 
     err = memif_get_details(conn, &md, buf, buflen);
     if (err != MEMIF_ERR_SUCCESS) {
-        log_info("%s", memif_strerror(err));
+//TBD        mesh_log(mc, MESH_LOG_INFO, "%s", memif_strerror(err));
```

Comment:
Not sure why you remove logging in many places in this file. Please make the logging working everywhere.